static/app/components/replays/replayDiff.tsx

```diff
@@ -0,0 +1,210 @@
+import {Fragment, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+import beautify from 'js-beautify';
```

Comment:
wherever we end up using this we'll still want to be lazy loaded since js-beautify and diff are pretty big

Reply:
ya for sure. It'll be on Issue Details, so I'll tap you again for that PR.

I think the lazy-load boundary will basically include the `useReplayData()` call, and then return this component to render. modulo there will be some wrapper divs/panels to make it look nice, but hopefully that's the extent of the lazy-loaded wrapper for the diff component.